<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Btn_Login">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(220, 138, 221);
background-color: rgb(87, 227, 137);</string>
   </property>
   <property name="text">
    <string>log in</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Txt_Name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>당신의 이름은?</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Txt_Pw">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Pw">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>비밀번호 4자리</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
